android/app-example/src/main/java/com/badoo/ribs/example/rib/hello_world/HelloWorldView.kt

```diff
@@ -35,11 +39,15 @@ class HelloWorldViewImpl private constructor(
     ) : this(context, attrs, defStyle, PublishRelay.create<Event>())
 
     override val androidView = this
+    lateinit var idText: TextView
```

Comment:
for the future, it is better to use lazy with vals

Reply:
Fixed